airbyte-integrations/connectors/source-yandex-metrica/source_yandex_metrica/schemas/download.json

```diff
@@ -0,0 +1 @@
+{}
```

Comment:
should this be empty?

Reply:
Yes. Download schema is different for `hits` and `visits`. Since Download stream passes its data to either Views or Sessions stream the schema gets dynamically generated there.